cmp/internal/diff/diff.go

```diff
@@ -93,9 +93,8 @@ type Result struct{ NumSame, NumDiff int }
 func BoolResult(b bool) Result {
 	if b {
 		return Result{NumSame: 1} // Equal, Similar
-	} else {
-		return Result{NumDiff: 2} // Not Equal, not Similar
 	}
+	return Result{NumDiff: 2} // Not Equal, not Similar
```

Comment:
This is deliberate.

Both branches have equal probability of being executed. They are kept at the same indentation for that purpose.